### ISSUE TITLE
toggle "Maximize"

### ISSUE THREAD

**Issue by rdewolff** [fikovnik/ShiftIt#173]
It would be great that the action "Maximize" would become "Toggle Maximize".

So activated once, it would switch the activate window from size A to max available size. 
Second activation would toggle max size to size A.

**Comment by Skinner927** [comment]
+1

**Comment by fikovnik** [comment]
I agree, but for this the windows would have to become stateful. This is long due, but hard to find time for this.
